Gemfile.lock

```diff
@@ -630,4 +630,4 @@ RUBY VERSION
    ruby 2.7.4p191
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```

Comment:
I've just noticed this change in bundler version. Will this cause an issue? Must have happened when I ran the `bundle install`. At first it didn't run correctly so I ran it with the `bundle update --bundler` command.

Reply:
It's all good Conor, we've been meaning to upgrade bundler anyway 😄